src/components/pre-playback-play-overlay/pre-playback-play-overlay.tsx

```diff
@@ -89,7 +90,8 @@ class PrePlaybackPlayOverlay extends Component<any, any> {
     if (!(props.prePlayback || isStartOver) || props.loading) {
       return undefined;
     }
-    const labelText = props.isPlaybackEnded ? props.startOverText : props.playText;
+    const entryName = `, ${this.props.title}: ${this.props.player.sources.metadata?.name}`;
```

Comment:
if you have the same exact text / same text calculation logic used twice, make it reusable.
either make a new component to render just the text, or put the computed string inside the redux state.
making a component is probably simpler.

Reply:
just for one line? is it not unnecessary?